conda-store-server/tests/test_actions.py

```diff
@@ -238,6 +248,18 @@ def test_add_lockfile_packages(
 def test_api_get_build_lockfile(
     request, conda_store, db, simple_specification_with_pip, conda_prefix, is_legacy_build, build_key_version
 ):
+    # sets build_key_version
+    if build_key_version == 0:  # invalid
+        with pytest.raises(TraitError, match=(
+            r"c.CondaStore.build_key_version: invalid build key version: 0, "
+            r"expected: \(1, 2\)"
+        )):
+            conda_store.build_key_version = build_key_version
+        return  # invalid, nothing more to test
+    conda_store.build_key_version = build_key_version
+    assert BuildKey.current_version() == build_key_version
+    assert BuildKey.versions() == (1, 2)
```

Comment:
This now initializes `build_key_version` via `conda_store` as one would in real life (via the config)